root_data.tf

```diff
@@ -54,3 +54,7 @@ data "github_ip_ranges" "actions_ranges" {}
 data "aws_ssm_parameter" "transform_engine_retry_role_arn" {
   name = "/${local.environment}/transform_engine/retry_role_arn"
 }
+
+data "aws_ssm_parameter" "transform_engine_v2_retry_role_arn" {
```

Comment:
This SSM parameter is the ARN for the SNS topic, so should change the resource name to: `transform_engine_v2_out_topic_arn`